server/views/partials/personalPage/careNeeds.njk

```diff
@@ -37,11 +37,14 @@
             {{ summaryListRow(need.type, careNeed(need), "care-need") }}
           {% endfor %}
         </dl>
-        {% if hasPastCareNeeds %}
+      {% endif %}
+      {% if not careNeeds.length and not hasPastCareNeeds %}
+        <p data-qa="no-care-needs-message">No care needs have been entered.</p>
+      {% elseif not careNeeds.length and hasPastCareNeeds %}
```

Comment:
Really frustrates me that nunjucks has its own language. Why cant they just use JS so I don't have to wonder if something is going to work?!